messenger.rst

```diff
@@ -1745,14 +1749,6 @@ in the table.
     The length of time to wait for a response when calling
     ``PDO::pgsqlGetNotify``, in milliseconds.
 
-The keepalive feature, which prevents messages from being prematurely redelivered during
-long-running processing, updates the ``delivered_at`` timestamp periodically to ensure
-the message is marked as "in progress".
```

Comment:
I don't think this is needed since no other transport has an explanation of how the keepalive mechanism is implemented.

Reply:
While this is an implementation detail, it could be helpful to understand the how, that said, its ok for me to remove it.

Reply:
Yes 👍

Reply:
Done, see #20736 for 7.2